cmake/SearchForStuff.cmake

```diff
@@ -111,3 +111,10 @@ else()
   set(IGN_MATH_VER ${ignition-math6_VERSION_MAJOR})
   message(STATUS "Looking for ignition-math${IGN_MATH_VER}-config.cmake - found")
 endif()
+
+########################################
+# Find ignition cmake2
+# Only for using the testing macros, not really
+# being use to configure the whole project
+find_package(ignition-cmake2 2.3 REQUIRED)
```

Comment:
How about checking `if (IGNITION-TOOLS_BINARY_DIRS)`?

Reply:
I don't think that is useful because we often install `ign-tools` in windows workspaces. I think we want the macros available in all cases, so I don't think we should conditionally find this